Example/PaymentSheet Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlayground.swift

```diff
@@ -15,6 +15,9 @@ import UIKit
 import SwiftUI
 import PassKit
 
+let endpointSelectorEndpoint = "https://stripe-mobile-payment-sheet-test-playground-v6.glitch.me/endpoints"
+let defaultCheckoutEndpoint = "https://stripe-mobile-payment-sheet-test-playground-v6.glitch.me/checkout"
+
```

Comment:
These would be better inside `PaymentSheetTestPlayground`, make them `static` so you can access the default one before init